src/style.css

```diff
@@ -1 +1,67 @@
-/* Styles go here */
+@font-face {
+  font-family: "Roboto";
+  src: url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  line-height: 0;
```

Comment:
why do you need it?

Reply:
To remove line-height in one link, I have already removed it.

Reply:
why do you need to remove the line-height here?)

Reply:
You are right, i don`t need to remove it)